common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -248,34 +248,6 @@ def test_bad_filename_error(self):
         self.import_page.upload_tarball('funny_cat_video.mp4')
         self.import_page.wait_for_filename_error()
 
-    def test_task_list(self):
```

Comment:
These are checkpoints related to import-- not checklists. This test should not be deleted.